test/integration/helpers/serverspec/shared_examples.rb

```diff
@@ -0,0 +1,24 @@
+require 'serverspec'
+
+set :backend, :exec
+
+shared_examples 'nfs' do
+  if os[:release].to_i < 7
+    describe service('nfs') do
+      it { should be_enabled }
+      it { should be_running }
+    end
+  else
+    describe service('nfs') do
+      it { should be_enabled }
+    end
```

Comment:
No `it { should be_running }`?

Reply:
On CentOS 7, systemd just runs and exits for the 'nfs' service so this fails. This was the primary reason I fixed the tests :)

Reply:
I'll add a comment here so that it's clear.

Reply:
Strange, it worked for me when I added it before, now it's failing when I tear down and start the VM...

**Found out why!** If you:
1) converge the recipe
2) run `systemctl status nfs` on the VM
3) run `kitchen verify default-centos-7`

The verify will work, somehow. I'm not sure how it magically changes the kitchen test, but this PR is correct.